easybuild/easyblocks/c/cp2k.py

```diff
@@ -103,6 +103,7 @@ def extra_options():
             'plumed': [None, "Enable PLUMED support", CUSTOM],
             'type': ['popt', "Type of build ('popt' or 'psmp')", CUSTOM],
             'typeopt': [True, "Enable optimization", CUSTOM],
+            'nvarch': ['sm_72', "NVidia architecture", CUSTOM],
```

Comment:
@Micket The default value should be "neutral", so I'd go with `None` (and check below whether any other value is specified before trying to use it)?

Reply:
Well, OK, but I would consider it basically an error to not set it (and CUDA is used). If not set, nvcc defaults to sm_30. Everyone building do want to set this, so that they aren't unwittingly getting a poor build.

Reply:
OK, then add a check to make sure it's set, which is better than using a default that'll get outdated in no time (and which we can't update easily without breaking backward compatibiltiy).